Tests/UnitTests/Plugins/ExaTrkX/CMakeLists.txt

```diff
@@ -4,3 +4,11 @@ add_unittest(ExaTrkXTensorConversion ExaTrkXTensorConversionTests.cpp)
 add_unittest(ExaTrkXEdgeBuilding ExaTrkXEdgeBuildingTest.cpp)
 add_unittest(ExaTrkXBoostTrackBuilding ExaTrkXBoostTrackBuildingTests.cpp)
 add_unittest(ExaTrkXMetricHookTests ExaTrkXMetricHookTests.cpp)
+if(ACTS_EXATRKX_ENABLE_CUDA)
+    add_unittest(ConnectedComponentsCuda ConnectedComponentCudaTests.cu)
+    # TODO this is not nice, not sure why it does not propagate through...
+    set_property(
+        TARGET ActsUnitTestConnectedComponentsCuda
+        PROPERTY CUDA_STANDARD 20
```

Comment:
Suggest adding a minimum CMake requirement of 3.18 to this file (https://cmake.org/cmake/help/latest/prop_tgt/CUDA_STANDARD.html)